metadata/labels.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+import "fmt"
+
+// Common constants
+const (
+	// rhtapDomain is the prefix of the application label
+	rhtapDomain = "appstudio.openshift.io"
```

Comment:
@scoheb https://github.com/redhat-appstudio/book/blob/main/ADR/0024-release-attribution.md#attribution-for-a-particular-release talks about `rhtap.openshift.io` as our "domain." Do we want the author and attribution labels to have this new rhtap domain or be the same as our other labels and use this appstudio domain?

Reply:
David is asking in slack...

Reply:
An ADR will be prepared to address this. We can leave it as it is for the moment.

Reply:
Okay, just pointing out in case it was missed, the author label uses this so this pr changes the author label domain from rhtap to appstudio. But we can align them all once the ADR is prepared